generators/client/templates/angular/webpack/_webpack.prod.js

```diff
@@ -65,6 +65,19 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
             ],
             exclude: ['node_modules/generator-jhipster']
         },
+        {
+            test: /\.ts$/,
+            use: [
+                {
+                    loader: 'ngc-webpack',
+                    options: {
+                        disable: false,
```

Comment:
@deepu105 But we have given `resourceoverride`in plugin with an empty value. we can remove it.